pylint/__pkginfo__.py

```diff
@@ -9,7 +9,7 @@
 
 from __future__ import annotations
 
-__version__ = "3.0.0a6"
+__version__ = "3.0.0b1"
```

Comment:
Should this be bumped _after_ releasing 3.0.0a6?

Reply:
There's three commits, we're going rebase and merge them, then tag the second one.